themes/courses.labster.com/lms/templates/emails/account_creation_and_enroll_emailMessage.txt

```diff
@@ -0,0 +1,22 @@
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+${_("Welcome to {course_name}").format(course_name=course.display_name_with_default_escaped)}
+
+${_("To get started, please visit https://{site_name}. The login information for your account follows.").format(site_name=site_name)}
```

Comment:
@lukyfebrianto is this text is approved by CS?

Reply:
@TamoshaytisV it's a default edX's template copy-pasted to the theme. Why do we need approval from CS?